src/components/BottomSheetInLineNotification.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
+import { StyleSheet } from 'react-native'
+import Animated, { useAnimatedStyle, useSharedValue } from 'react-native-reanimated'
+import InLineNotification, { InLineNotificationProps } from 'src/components/InLineNotification'
+import Modal from 'src/components/Modal'
+import { useShowOrHideAnimation } from 'src/components/useShowOrHideAnimation'
+import { Spacing } from 'src/styles/styles'
+
+interface Props extends InLineNotificationProps {
+  showNotification: boolean
+}
+
+// this value is used to ensure the notification is offset by its own height when transitioning in and out of view
+const NOTIFICATION_HEIGHT = 200
+
+// for now, this notification is launched from the bottom of the screen only
+const BottomSheetInLineNotification = ({ showNotification, ...inLineNotificationProps }: Props) => {
+  const [isVisible, setIsVisible] = useState(showNotification)
+
+  const progress = useSharedValue(0)
+  const animatedStyle = useAnimatedStyle(() => {
```

Comment:
looks like if we get rid of `animatedStyle` nothing will change. 
my guess it's because animation is in fact handled by `Modal`.

Reply:
i'm not sure that's true - the animatedStyle here is doing the up / down animation for the notification. without this, the notification will fade in / out only